apps/meteor/client/views/room/composer/ComposerOmnichannel/ComposerOmnichannelInquiry.tsx

```diff
@@ -32,11 +36,24 @@ export const ComposerOmnichannelInquiry = (): ReactElement => {
 		}
 	};
 
-	const t = useTranslation();
+	const title = useMemo(() => {
+		if (user?.status === 'offline') {
+			return t('meteor_status_offline');
+		}
+
+		if (!agentAvailable) {
+			return t('Youre_unavailable');
```

Comment:
i think there is no need for two different reasons, not being available is valid for both cases

Reply:
I think there's some value on diff. I was considering this being a message for agents so being explicit with what they have to do may be better as these types of users may not have any familiarity with RC.

So giving them clear feedback may help them to fix "faster"

Reply:
If this is the case, considering lay users the tool, the message should not be: "you can't take it because you are offline" instead of "you are offline" ??

Reply:
I'm totally fine with that!

@tiagoevanp hehe